esp32s2-hal/examples/systimer.rs

```diff
@@ -41,8 +41,8 @@ fn main() -> ! {
 
     let syst = SystemTimer::new(peripherals.SYSTIMER);
 
```

Comment:
Could you add the print statement from the other examples here please?

```suggestion
esp_println::println!("SYSTIMER Current value = {}", SystemTimer::now());
```